videoretrieval/datasets/msrvtt/constants.py

```diff
@@ -25,4 +26,40 @@
 train_val_zip_path = base_dir / "train_val_zip_metadata.zip"
 train_val_json_filename = "train_val_videodatainfo.json"
 test_json_path = base_dir / "test_metadata.json"
-processed_metadata_path = base_dir / "metadata.json"
\ No newline at end of file
+processed_metadata_path = base_dir / "metadata.json"
+
+features_tar_url = "http://www.robots.ox.ac.uk/~vgg/research/collaborative-experts/data/features-v2/MSRVTT-experts.tar.gz"
```

Comment:
We can ask Arka to see if he has any suggestions about using/organizing constants.